SwanGallery/src/handler.ts

```diff
@@ -0,0 +1,43 @@
+import { URLExt } from '@jupyterlab/coreutils';
+import { ServerConnection } from '@jupyterlab/services';
+
+/**
+ * Calls /api/contents/fetch/ with the url to download,
+ * which downloads the notebook/project URL to the home directory of the user 
+ * @returns The Jupyter server contents API response of the downloaded file
+ */
+
+export async function downloadUrlFromServer(
```

Comment:
Why is this helper function defined in a different file than the other helper `initConfigurationFromServer` and the main function they are used from?